packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -74,8 +74,8 @@ const chatMessageStyles: ComponentSlotStylesInput<
     marginBottom: v.headerMarginBottom,
   }),
 
-  content: ({ variables: v }): ICSSInJSStyle => ({
-    display: 'block',
+  content: ({ props: p, variables: v }): ICSSInJSStyle => ({
+    display: p.reactions && p.reactionsPosition === 'end' ? 'inline-block' : 'block',
```

Comment:
I had to conditionally change the display prop on the content, because if the reactionsPosition is 'end' in Teams theme, the reactions should appear after the content. No better idea in the moment (tried added float elements, but then the paddings are broken...) I didn't want to create additional element for wrapping this, so this seemed like the only option.